Home.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
```

Comment:
This file was renamed to `index.html` in the commit 4bb0e3b076376ec303be6ee78b9f3bd9531288a4.
So you should edit `index.html` instead of adding this file.
(But the change to this file looks unnecessary...)

Reply:
yes, thanks for informing. 
There is no change on this file comparing with the prior version, so please instead it by the index.html, 
I will notice it the next time.